src/components/ReviewComponents/utils.ts

```diff
@@ -11,13 +11,15 @@ const CPUResourceMap = {
   '': 'cores',
 };
 
-export const createResourceData = ({ memory = '', cpu = '' }) => {
+export const createResourceData = (resources) => {
```

Comment:
Nit: Would be nice to have a type for `resources`.

Reply:
Done.